code/datums/looping_sounds/machinery_sounds.dm

```diff
@@ -182,3 +182,19 @@
 	end_volume = 60
 	extra_range = MEDIUM_RANGE_SOUND_EXTRARANGE
 	falloff_exponent = 4
+
+/datum/looping_sound/cryo_cell
```

Comment:
```php
/datum/looping_sound/proc/play(soundfile, volume_override)
	var/sound/sound_to_play = sound(soundfile)
	if(direct)
		sound_to_play.channel = sound_channel || SSsounds.random_available_channel()
		sound_to_play.volume = volume_override || volume //Use volume as fallback if theres no override
		SEND_SOUND(parent, sound_to_play)
	else
		playsound(
			parent,
			sound_to_play,
			volume,
			vary,
			extra_range,
			falloff_exponent = falloff_exponent,
			pressure_affected = pressure_affected,
			ignore_walls = ignore_walls,
			falloff_distance = falloff_distance,
			use_reverb = use_reverb
		)
```
`#define SOUND_RANGE 17`
`var/maxdistance = SOUND_RANGE + extrarange`
`listeners = get_hearers_in_view(maxdistance, turf_source)`

if you're not specifying the range for your looping sound, then it'll be the 17 (the default), I think 17 is pretty high.

Reply:
You aren't factoring in the sound falloff exponent and distance. Even if those aren't set it still gets applied.

```
	for(var/mob/listening_mob in listeners | SSmobs.dead_players_by_zlevel[source_z])//observers always hear through walls
		if(get_dist(listening_mob, turf_source) <= maxdistance)
			listening_mob.playsound_local(turf_source, soundin, vol, vary, frequency, falloff_exponent, channel, pressure_affected, S, maxdistance, falloff_distance, 1, use_reverb)
			. += listening_mob
```

![Code_sZTquqcAJo](https://github.com/user-attachments/assets/e8fb6d7a-bea4-4c66-9cb2-e628cb6662b4)

Falloff_disance is 6 and falloff_exponent is 1.

But on debugging there are a lot of sounds that are getting sent to mobs that shouldn't be within hearing distance due to the falloff. It's basically playing sounds with no volume... which is baaaaaaad.